src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  const threeDayDiscount = 20;
+  const sevenDayDiscount = 50;
```

Comment:
what if we will change days amount to another numbers - we will need to change these variables names too
try to avoid numbers in variables names